pom.xml

```diff
@@ -115,6 +116,7 @@
     <commons.logging.version>1.1.1</commons.logging.version>
     <commons.cli.version>1.3.1</commons.cli.version>
     <shiro.version>1.3.2</shiro.version>
+    <joda.version>2.9.9</joda.version>
```

Comment:
does this get propagated to profile?
spark 2.3 is built with 2.9.3 - would it conflict?
https://github.com/apache/spark/blob/branch-2.3/pom.xml#L175

Reply:
it would not, it is only used in zeppelin-zengien module